src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -74,8 +74,8 @@ private function configureRoutes(RoutingConfigurator $routes): void
     {
         $configDir = $this->getConfigDir();
 
-        $routes->import($configDir.'/{routes}/'.$this->environment.'/*.{php,yaml}');
-        $routes->import($configDir.'/{routes}/*.{php,yaml}');
+        $routes->import($configDir.'/{routes}/'.$this->environment.'/*.{php,xml,yaml}');
+        $routes->import($configDir.'/{routes}/*.{php,xml,yaml}');
 
         if (is_file($configDir.'/routes.yaml')) {
```

Comment:
If I'm not mistaking shouldn't this case be covered as well?

Reply:
new feature IMHO, previous PR explicitly targeted config/packages. I follow :)

changelog updated :+1:

Reply:
sorry, i missed the else case.

@nicolas-grekas WDYT?

Reply:
ideally i'd like this to be simple as `$routes->import($configDir.'/routes.{php,xml,yaml}');`

for services we could phase out services_<env>.ext in favor `when@<env>` notation IIUC.

Then everything is uniform IMHO.